_posts/2018-05-15-2017-recap.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "2017 Reflections &#124; Projections: A Recap"
+date:   2018-05-15 12:00:00
+image: "/assets/images/squiggles2.png"
+categories: spotlight
+tags: rp2018
```

Comment:
can you also add the tag `featured` here and remove it from the previous blog post -- that way we only feature the latest post!